openmetadata-ui/src/main/resources/ui/src/components/Glossary/GlossaryTerms/tabs/AssetsTabs.interface.ts

```diff
@@ -26,7 +26,7 @@ export enum AssetsOfEntity {
 }
 
 export interface AssetsTabsProps {
-  onAddAsset: () => void;
+  onAddAsset?: () => void;
```

Comment:
this is not needed now, we are going to have onAddAsset

Reply:
Updated